theano/gpuarray/tests/test_abstractconv.py

```diff
@@ -196,7 +196,7 @@ def setup_class(cls):
         cls.shared = staticmethod(gpuarray_shared_constructor)
         cls.mode = mode_with_gpu.excluding('cudnn')
 
-    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1, 1)):
+    def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1, 1), u=False):
```

Comment:
For the 3D case, I've put a [check](https://github.com/Theano/Theano/pull/6007/files#diff-3aa5dca5e3dfdb813cd63e62f174bf06R383) in `run_fwd` to SkipTest if it is not a 2D convolution.